templates/project/config.ru

```diff
@@ -1,7 +1,8 @@
 require 'dashing'
+require 'securerandom'
 
 configure do
-  set :auth_token, 'YOUR_AUTH_TOKEN'
+  set :auth_token, ENV['AUTH_TOKEN'] || SecureRandom.uuid
```

Comment:
There *are* subtle differences...

```
@y = 0
def y
  @y += 1
end
```

Example                 | X is unset | X is anything | X is nil | unset, second call
------------------------|------------|---------------|----------|-------------------
`ENV["X"] || y`         | 1          | {anything}    | 1        | 2
`ENV.fetch("X") { y }`  | 1          | {anything}    | nil      | 2
`ENV.fetch("X", y)`     | 1          | {anything}    | nil      | 1